lexicons/com/atproto/admin/sendEmail.json

```diff
@@ -9,11 +9,12 @@
         "encoding": "application/json",
         "schema": {
           "type": "object",
-          "required": ["recipientDid", "content"],
+          "required": ["recipientDid", "content", "senderDid"],
           "properties": {
             "recipientDid": { "type": "string", "format": "did" },
             "content": { "type": "string" },
-            "subject": { "type": "string" }
+            "subject": { "type": "string" },
+            "senderDid": { "type": "string", "format": "did" }
```

Comment:
maybe `creatorDid` or `requesterDid` is clearer? this don't impact who the email appears to come from for the recipient, just what is recorded in the mod event (I assume)

Reply:
hmm... idk, neither of those sound better/clearer than `senderDid` in the context tbh.